packages/ckeditor5-minimap/package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "@ckeditor/ckeditor5-minimap",
+  "version": "0.0.1",
+  "description": "Content minimap feature for CKEditor 5.",
+  "keywords": [
+    "ckeditor",
+    "ckeditor5",
+    "ckeditor 5",
+    "ckeditor5-feature",
+    "ckeditor5-plugin",
+    "ckeditor5-dll"
+  ],
+  "main": "src/index.js",
+  "dependencies": {
+    "ckeditor5": "^29.0.0"
+  },
+  "devDependencies": {
+    "@ckeditor/ckeditor5-alignment": "^29.0.0",
+    "@ckeditor/ckeditor5-basic-styles": "^29.0.0",
+    "@ckeditor/ckeditor5-cloud-services": "^29.0.0",
+    "@ckeditor/ckeditor5-code-block": "^29.0.0",
+    "@ckeditor/ckeditor5-core": "^29.0.0",
+    "@ckeditor/ckeditor5-dev-utils": "^25.0.0",
+    "@ckeditor/ckeditor5-easy-image": "^29.0.0",
+    "@ckeditor/ckeditor5-editor-decoupled": "^29.0.0",
+    "@ckeditor/ckeditor5-font": "^29.0.0",
+    "@ckeditor/ckeditor5-image": "^29.0.0",
+    "@ckeditor/ckeditor5-indent": "^29.0.0",
+    "@ckeditor/ckeditor5-page-break": "^29.0.0",
+    "@ckeditor/ckeditor5-table": "^29.0.0",
+    "@ckeditor/ckeditor5-theme-lark": "^29.0.0"
+  },
+  "engines": {
+    "node": ">=12.0.0",
+    "npm": ">=5.7.1"
+  },
+  "author": "CKSource (http://cksource.com/)",
+  "license": "GPL-2.0-or-later",
+  "homepage": "https://ckeditor.com/ckeditor-5",
+  "bugs": "https://github.com/ckeditor/ckeditor5/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ckeditor/ckeditor5.git",
+    "directory": "packages/ckeditor5-minimap"
+  },
+  "files": [
+    "lang",
+    "src",
+    "theme"
```

Today I merged https://github.com/ckeditor/ckeditor5/pull/10093 which adds a new entry in the `files` array.

```suggestion
    "theme",
    "ckeditor5-metadata.json"
```